static/monorail-2.0.yaml

```diff
@@ -4680,6 +4680,39 @@ paths:
       - Write
       - workflowsModify
     x-swagger-router-controller: workflows
+  /notification/alerts:
```

Comment:
I also have concern for southbound and northbound API merge. some comments are updated to https://rackhd.atlassian.net/browse/RAC-1447